src/ngraph/op/fused/lstm_cell.hpp

```diff
@@ -74,27 +73,32 @@ namespace ngraph
             public:
                 static constexpr NodeTypeInfo type_info{"LSTMCell", 0};
                 const NodeTypeInfo& get_type_info() const override { return type_info; }
-                LSTMCell() = default;
+                LSTMCell()
```

Comment:
This implementation belongs in the the cpp file

Reply:
I don't see the default constructor definition in the cpp file, but for convenience, I have split the declaration and the definition in the next commit.